StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/ScriptMessageHandlerWithReply.swift

```diff
@@ -36,8 +36,8 @@ class ScriptMessageHandlerWithReply<Payload: Decodable, Response: Encodable>: NS
 
             return (response, nil)
         } catch {
-            debugPrint("Error processing message: \(error.localizedDescription)")
-            return (nil, error.localizedDescription)
+            debugPrint("Error processing message: \((error as NSError).debugDescription)")
+            return (nil, (error as NSError).debugDescription)
```

Comment:
I think the debug description is more valuable than the user-facing localized description for logging in the web layer. For errors that don't have a localized description implemented, this was just showing the error domain & code, but I believe that most NSError implementations have this encoded in the debugDescription along with some other useful things.

Reply:
Note: I originally made this change because I was playing around with making `openAuthenticatedWebView` an async function that returned a debug error message to the web view, but ended up reverting that change. But I think we till want the debug description here anyway so leaving it in.